src/Connector.sol

```diff
@@ -334,4 +334,50 @@ contract CentrifugeConnector {
         require(token.hasMember(destinationAddress), "CentrifugeConnector/not-a-member");
         token.mint(destinationAddress, amount);
     }
+
+    function handleExecutedDecreaseInvestOrder(
+        uint64 poolId,
+        bytes16 trancheId,
+        bytes32 investor,
+        uint128 currency,
+        uint128 currencyPayout,
+        uint128 remainingInvestOrder
+    ) public onlyGateway {
+        // TODO: Implement
```

Comment:
Do we still want to implement this here or leave it for a follow up PR when Alina's Pr is merged? Or base this one off Alina's and do it right away?

Reply:
yeah, I agree. Can be removed :) I have added the implementation in my pr